source/fundamentals/collations.txt

```diff
@@ -4,5 +4,302 @@ Collations
+have specific character ordering conventions that differ from the ASCII
+standard.
+
+For example, in Canadian French, the right-most accented character determines
+the ordering for strings when the other characters are the same. Consider the
+following Canadian French words: 
+
+- cote
+- coté
+- côte
+- côté
+
+When using the default binary collation, MongoDB sorts them in the following order:
+
+.. code-block:: none
+
+   cote
+   coté
+   côte
+   côté
+
+When using the Canadian French collation, MongoDB sorts them in the following order:
+
+.. code-block:: none
+
+   cote
+   côte
+   coté
+   côté
+
+Specify a Collation
+-------------------
+
+To create a Collation object, you must specify a ``Locale`` field; all other fields are
+optional. For example, the following code snippet specifies a Collation object with
+the ``"en_US"`` locale collation:
+
+.. code-block:: go
+
+   myCollation := &options.Collation{Locale: "en_US"}
+
+For a complete list of collation fields, visit the `Collation API documentation 
+<{+api+}/mongo/options#Collation>`__. To see all the supported locales and the
+default values for the ``Locale`` fields, visit :manual:`Supported Languages and Locales
+</reference/collation-locales-defaults/#supported-languages-and-locales>`.
+
+Usage
+-----
+
+You can specify a collation when you create a new collection, view, or index. You can also
+specify a collation in a subset of operations.
+
+Collection or View
+~~~~~~~~~~~~~~~~~~
+
+You can specify a collation when you create a new collection or view. This defines the default
+collation for any operations called on that collection or view. Set a collation through a 
+``CreateCollectionOptions`` or ``CreateViewOptions`` object. Then, call the
+``CreateCollection()`` or ``CreateView()`` method with your options object as an argument.
+
+Index
+~~~~~
+
+You can specify a collation when you create a new index on a collection. The index stores
+an ordered representation of the documents in the collection so your operation does not need
+to perform the ordering in-memory. 
+
+To use the index in an operation, your operation must specify the same collation as the one
+specified in the index. Additionally, ensure that the operation is covered by the index that
+contains the collation. Set a collation through an ``IndexOptions`` object and call the
+``CreateOne()`` method with your options object as an argument.
+
+Operation
+~~~~~~~~~
+
+Specifying a collation on an operation overrides any default collation previously
+defined for a collection. If you specify a new collation on an operation that differs
+from the index collation, you cannot use that index. As a result, the operation may not
+perform as well as one that is covered by an index. 
+
+For more information on the disadvantages of sorting operations not covered by an index,
+see the server manual page on :manual:`Using Indexes to Sort Query Results </tutorial/sort-results-with-indexes/>`.
```

Comment:
Issue: I believe the term "server manual" is an internal distinction more than an external distinction. I believe readers generally just see our docs as "MongoDB documentation". I'd avoid using the term "server manual" as a reader likely won't know what this means and won't gain much value from this being stated.

```suggestion
see :manual:`Using Indexes to Sort Query Results </tutorial/sort-results-with-indexes/>`.
```

Reply:
Took out the reference to the server manual